### PR TITLE
Avatar and Badge - props deprecation

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
Avatar and Badge - props deprecation.

## Changelog
Avatar and Badge - props deprecation.

### CODE REVIEW

File: generatedTypes/src/components/badge/index.d.ts [hunk 1]
```diff
@@ -24,7 +15,7 @@ export declare type BadgeProps = ViewProps & TouchableOpacityProps & {
     /**
      * the badge size (default, small)
```
Comment (mendyEdri) [inline generatedTypes/src/components/badge/index.d.ts:16]:
```suggestion
     * the badge size (default - 6)
```

File: generatedTypes/src/components/badge/index.d.ts [hunk 2]
```diff
@@ -357,7 +343,7 @@ declare const _default: React.ComponentClass<ViewProps & TouchableOpacityProps &
...
      */
-    size?: number | "default" | "small" | "pimpleSmall" | "pimpleBig" | "pimpleHuge" | "large" | undefined;
+    size?: number | undefined;
```
Comment (mendyEdri) [inline generatedTypes/src/components/badge/index.d.ts:346]:
```suggestion
    size?: number;
```

**Review by mendyEdri — COMMENTED** [review]
I would add internal enum for the numbers (default, small, etc') 
let me know if you need help with it

File: generatedTypes/src/components/badge/index.d.ts (hunk 2 above)
Comment (Inbal-Tish) [reply generatedTypes/src/components/badge/index.d.ts:346]: I just deleted the enum, we want to support only number

File: generatedTypes/src/components/badge/index.d.ts (hunk 1 above)
Comment (Inbal-Tish) [reply generatedTypes/src/components/badge/index.d.ts:16]: I fixed the description

**Review by mendyEdri — APPROVED** [review]

**Pull request merged**